### ISSUE TITLE
Tead not working with node v.10 (OSX 10.13.3)

### ISSUE THREAD

**Issue by mindreframer** [okwolf/tead#19]
Hey, just wanted to share that i could not bring tead to run its own tests.... Maybe this is fixable. Not a user currently, wanted just to take a look

```
tead|master ✗⇒ npm t

> tead@0.4.3 pretest /Users/roman/Desktop/teadtest/tead
> node test/bootstrap


> tead@0.4.3 test /Users/roman/Desktop/teadtest/tead
> node src/tead.js --coverage

node[27722]: ../src/node_file.cc:829:void node::fs::Stat(const FunctionCallbackInfo<v8::Value> &): Assertion `(argc) == (3)' failed.
 1: node::Abort() [/usr/local/bin/node]
 2: node::Assert(char const* const (*) [4]) [/usr/local/bin/node]
 3: node::fs::LStat(v8::FunctionCallbackInfo<v8::Value> const&) [/usr/local/bin/node]
 4: v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [/usr/local/bin/node]
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/usr/local/bin/node]
 6: v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/usr/local/bin/node]
 7: 0x1c37e1d0427d
 8: 0x1c37e1d144f7


tead|master ✗⇒ node -v
v10.0.0

```

**Comment by okwolf** [comment]
I updated the CI config to include Node.js 10 and was able to [reproduce the issue](https://travis-ci.org/teadjs/tead/jobs/406115840#L455). No fix has been identified yet.

Thanks for pointing this out.

**Comment by okwolf** [comment]
@mindreframer good news! I found what seems to be a [related bug](https://github.com/GoogleChrome/workbox/issues/1450) and have applied a [fix now](https://travis-ci.org/teadjs/tead/builds/406138245).

🎉

**Comment by mindreframer** [comment]
Awesome! Will check it later! Thx!

**Comment by okwolf** [comment]
@mindreframer I've released [0.4.4](https://github.com/teadjs/tead/releases/tag/0.4.4). Please have a look.
